src/test/kotlin/no/nav/hjelpemidler/http/slack/SlackTest.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.hjelpemidler.http.slack
+
+/* import kotlinx.coroutines.runBlocking
```

Comment:
Ikke noe fan av utkommentert kode, spesielt ikke  i kode som flere jobber med. Siden denne bare er for lokal testing sikkert vill jeg bare markert klassen med:
`@Ignore("Brukes kun til å teste Slack-integrasjon under utvikling")` fra kotlin.test.Ignore

Reply:
Nice, det funker.